sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuSemaphore.scala

```diff
@@ -271,7 +271,7 @@ object GpuSemaphore {
     }
   }
 
-  val DEFAULT_PRIORITY: Long = 0L
+  val DEFAULT_PRIORITY: Long = -1L
```

Comment:
why this change?

Reply:
That is a good point. I originally was thinking this was for something else, but now that I look more deeply it is a bit of a hack. I need to go back and make it more explicit.